doc/tutorials/transformer/token_classification.ipynb

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
<p>Some introductory text about the cells below would again be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good call.</p>